src/Frontend.hs

```diff
@@ -68,11 +68,12 @@ runFrontend cfg = do
     app <- serveFAction (Proxy :: Proxy AulaActions) stateProxy extendClearanceOnSessionToken
             runAction aulaActions
 
-    unNat persist genInitialTestDb -- FIXME: Remove Bootstrapping DB
+    Right _ <- runExceptT $ unNat persist genInitialTestDb -- FIXME: Remove Bootstrapping DB
```

Comment:
This could be defined in a function, saving lot of boilerplate.

````haskell
runPersist :: (Persist :~> ExceptT e IO) -> Persist a -> IO a
runPersist persist m = either (error . show) id <$> runExceptT (unNat persist) m
````

This helper could be used in the Test Suite and here, as this snippet is going to be removed soon.